README.md

```diff
@@ -3,18 +3,24 @@
 ![](https://visitor-badge.laobi.icu/badge?page_id=arbiter)
 ![Github Actions](https://github.com/primitivefinance/arbiter/workflows/Rust/badge.svg)
 
-> A stand-alone Rust analytic program for UniswapV3 pools and...
+> A stand-alone analytic program for UniswapV3 pools wirtten in rust.
```

Comment:
minor spelling mistake here

Reply:
nice catch. Spelling is not my strong suit. Hence why i gravitate to math.

Reply:
love it - no worries

Reply:
love it - no worries